core/templates/dev/head/pages/landing/fractions/landing_page_teacher.html

```diff
@@ -335,54 +368,113 @@ <h2 class="oppia-fractions-landing-h2 oppia-fractions-landing-centered-h2" style
     .oppia-fractions-landing-image {
       position: relative;
       z-index: 20;
-      max-width: 80%;
+      max-width: 100%;
       max-height: 500px;
     }
     .oppia-fractions-landing-image-mobile {
       display: none;
     }
+    .video-frame {
+      position: relative;
```

Comment:
For each CSS rule please alphabetize (e.g. background then border then border-bottom-width etc.) Ditto below.